docs/source/conf.py

```diff
@@ -44,12 +44,16 @@
     "sphinxcontrib.contentui",
 ]
 
+# Add the numpydoc_show_inherited_class_members option
+numpydoc_show_inherited_class_members = False
```

Comment:
This might fix the issues we had with `TypedDict` too

Reply:
it may be... @ElliottKasoar